cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -19,10 +19,12 @@ module Cardano.DbSync (
   runDbSyncNode,
   runDbSync,
   -- For testing and debugging
-  FetchError (..),
+  OffChainFetchError (..),
   SimplifiedOffChainPoolData (..),
   httpGetOffChainPoolData,
-  parsePoolUrl,
+  parseOffChainPoolUrl,
```

Comment:
These are most probably imported and reexporeted for tools like http-get-json-metadata.hs. But can't these tools directly import the underlying modules?